python/uv.lock

```diff
@@ -80,7 +80,7 @@ wheels = [
 
 [[package]]
 name = "feldera"
-version = "0.30.0"
+version = "0.31.0"
```

Comment:
Is this something that should get updated automatically as part of the release?

Reply:
Actually, I am not sure what the protocol is for the lock file. We bump the version in the `pyproject.toml` and this lock file should get updated automatically when we use it.

Perhaps it makes sense to update the lock file as well.